### ISSUE TITLE
Compatibility issue with Laravel 8

### ISSUE THREAD

**Issue by guyaumetremblay** [bugsnag/bugsnag-laravel#464]
### Describe the bug
Hi!
I'm getting this dependencies issue when I attempt to add the library to a Laravel 8 project: 
```
-> composer require bugsnag/bugsnag-laravel
Using version ^2.22 for bugsnag/bugsnag-laravel
./composer.json has been updated
Running composer update bugsnag/bugsnag-laravel
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - bugsnag/bugsnag-laravel[v2.22.0, ..., v2.22.2] require bugsnag/bugsnag-psr-logger ^1.4 -> satisfiable by bugsnag/bugsnag-psr-logger[v1.4.0, ..., 1.5.x-dev (alias of dev-master)].
    - bugsnag/bugsnag-psr-logger 1.5.x-dev is an alias of bugsnag/bugsnag-psr-logger dev-master and thus requires it to be installed too.
    - bugsnag/bugsnag-psr-logger[dev-master, v1.4.0, ..., v1.4.3] require psr/log ^1.0 -> found psr/log[1.0.0, ..., 1.1.4] but the package is fixed to 2.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires bugsnag/bugsnag-laravel ^2.22 -> satisfiable by bugsnag/bugsnag-laravel[v2.22.0, v2.22.1, v2.22.2].
```

Seems to be a problem with version 2.0.0 of `psr/log`, which came with symfony. 

### Steps to reproduce
1. start a new project: `curl -s "https://laravel.build/example-app" | bash`
2. navigate to this new project directory: `cd example-app`
3. add a dependencies on bugsnag-laravel : `composer require bugsnag/bugsnag-laravel`
3.1 this step should failed 

### Environment
* Bugsnag version: latest
* PHP version: 8.0.8
* Composer version: 2.0.8
* Laravel version: 8

**Comment by GrahamCampbell** [comment]
If you manually add bugsnag and run composer update, it should work. Composer is not smart enough to downgrade the psr/log package to the 1.x line.

**Comment by guyaumetremblay** [comment]
@GrahamCampbell Thanks for your suggestion. This solution works, but I consider it as a workaround as it also update all my dependencies, which is unwanted.

**Comment by GrahamCampbell** [comment]
I suppose if you ran `composer require psr/log:^1.1` first, and then the require for bugsnag, it might work?

**Comment by guyaumetremblay** [comment]
@GrahamCampbell Yes it works. I did think about this solution but I was not sure if downgrading `psr/log` was a good idea.
Do you plan to support `psr/log` 2.0?

**Comment by mattdyoung** [comment]
We're looking at adding support for PSR v2 and v3 fairly soon. We'll provide an update here when that is available. Thanks!

**Comment by luke-belton** [comment]
Hi @guyaumetremblay - just to let you know we've [added support](https://github.com/bugsnag/bugsnag-psr-logger/pull/46) for PSR Log v2, this was released in [`v1.4.4`](https://github.com/bugsnag/bugsnag-psr-logger/releases/tag/v1.4.4) 🎉
